### PR TITLE
Foundation: `Process` is unavailable on android

### PR DESCRIPTION
**Pull request by compnerd**
Android does not provide Process (NSTask) like iOS.  Do not reference it
in such situations.

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @millenomi

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by buttaface** [comment]
Or you could get #2228 merged, after which the Android portion of this pull is unnecessary.

File: Foundation/NSObjCRuntime.swift [hunk 1]
```diff
@@ -255,75 +255,80 @@ internal let _NSClassesRenamedByObjCAPINotesInNetworking: [(swiftName: String, o
...
+        (UnitIlluminance.self, "NSUnitIlluminance"),
+        (UnitLength.self, "NSUnitLength"),
+        (UnitMass.self, "NSUnitMass"),
+        (UnitPower.self, "NSUnitPower"),
+        (UnitPressure.self, "NSUnitPressure"),
+        (UnitSpeed.self, "NSUnitSpeed"),
+        (UnitVolume.self, "NSUnitVolume"),
+        (UnitTemperature.self, "NSUnitTemperature"),
+    ]
+#if !(os(iOS) || os(Android)
```
Comment (spevans) [inline Foundation/NSObjCRuntime.swift]: Missing trailing `)`

**Comment by drodriguez** [comment]
I would prefer for someone to give another review to #2228 instead of merging this. In any case, after the change proposed by Simon, this would work.

File: Foundation/NSObjCRuntime.swift (hunk 1 above)
Comment (compnerd) [reply Foundation/NSObjCRuntime.swift]: Clearly pulling changes between two repositories is a great idea 🙄

**Comment by compnerd** [comment]
Well, for the iOS target, this will still be required.  I think that if that is finalised and merged first, then I can remove the android clause, otherwise, if this gets merged first, you could just remove the android clause after a rebase.

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test

File: Foundation/NSObjCRuntime.swift [hunk 2]
```diff
@@ -255,75 +255,80 @@ internal let _NSClassesRenamedByObjCAPINotesInNetworking: [(swiftName: String, o
...
-    (UnitLength.self, "NSUnitLength"),
-    (UnitMass.self, "NSUnitMass"),
-    (UnitPower.self, "NSUnitPower"),
-    (UnitPressure.self, "NSUnitPressure"),
-    (UnitSpeed.self, "NSUnitSpeed"),
-    (UnitVolume.self, "NSUnitVolume"),
-    (UnitTemperature.self, "NSUnitTemperature"),
-]
+internal let _NSClassesRenamedByObjCAPINotes: [(class: AnyClass, objCName: String)] = {
+    let map: [(AnyClass, String)] = [
```
Comment (drodriguez) [inline Foundation/NSObjCRuntime.swift]: I thing the compiler is telling you that you have to change this to `var` to be able to `append` later.

Comment (compnerd) [reply Foundation/NSObjCRuntime.swift]: haha, yeah, oops

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test

**Pull request merged**